tokio/src/runtime/builder.rs

```diff
@@ -554,6 +590,65 @@ impl Builder {
         self
     }
 
+    /// Sets the number of scheduler ticks after which the scheduler will poll the global
+    /// task queue.
+    ///
+    /// A scheduler "tick" roughly corresponds to one `poll` invocation on a task. Schedulers
+    /// have a local queue of already-claimed tasks, and a global queue of incoming tasks.
+    ///
+    /// Setting the interval to a smaller value increases the fairness of the scheduler,
+    /// at the cost of more synchronization overhead. That can be beneficial for prioritizing
+    /// getting started on new work, especially if tasks frequently yield rather than complete
+    /// or await on further I/O. Conversely, a higher value prioritizes existing work, and
+    /// is a good choice when most tasks quickly complete polling.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use tokio::runtime;
+    ///
+    /// # pub fn main() {
```

Comment:
This empty line shows up on the documentation. We should either remove the empty line or unhide the import.

(The same applies to `event_interval`, and apparently also the existing `thread_keep_alive`)

Reply:
Actually, it seems like it happens on quite a few methods on the `Builder`. It might be better to leave for another PR.

Reply:
Fixed in https://github.com/tokio-rs/tokio/pull/4671/commits/819998cffcf0f35d2b3affa323d06a85745f9182